platform-service/src/unstract/platform_service/helper/cost_calculation.py

```diff
@@ -30,21 +30,29 @@ def __init__(
         self.file_path = file_path
 
         if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
-            self.file_storage, self.file_path = self.__get_storage_credentials()
+            self.file_storage, self.file_path = self._get_storage_credentials()
         self.model_token_data = self._get_model_token_data()
 
     if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
 
-        def __get_storage_credentials(self) -> tuple[PermanentFileStorage, str]:
+        def _get_storage_credentials(self) -> tuple[PermanentFileStorage, str]:
             try:
                 # Not creating constants for now for the keywords below as this
                 # logic ought to change in the near future to maintain unformity
                 # across services
                 file_storage = json.loads(os.environ.get("FILE_STORAGE_CREDENTIALS"))
                 provider = FileStorageProvider(file_storage["provider"])
-                credentials = file_storage["credentials"]
+                if "credentials" in file_storage:
+                    credentials = file_storage["credentials"]
+                else:
+                    credentials = {}
```

Comment:
`.get` with `default` can be resolved this `if-else` condition.

Reply:
Thank you, Done.